internal/fs/fs.go

```diff
@@ -1692,7 +1692,14 @@ func (fs *fileSystem) Rename(
 		}
 	}
 
-	// Find the object in the old location.
+	// If child is a local file inode (un-synced), rename operation is not supported.
```

Comment:
child is confusing here, Just say the object to be renamed is a local file inode, rename operation is not supported.